binds/python/bind_misc.cpp

```diff
@@ -109,13 +109,13 @@ void bind_misc(py::module &m) {
 
 
     py::class_<morphio::Points>(m, "Points", py::buffer_protocol())
-        .def_buffer([](morphio::Points &m) -> py::buffer_info {
+        .def_buffer([](morphio::Points &points) -> py::buffer_info {
                 return py::buffer_info(
-                    m.data(),                               /* Pointer to buffer */
+                    points.data(),                               /* Pointer to buffer */
                     sizeof(float),                          /* Size of one scalar */
                     py::format_descriptor<float>::format(), /* Python struct-style format descriptor */
                     2,                                      /* Number of dimensions */
-                    { (int)  m.size(), 3 },                 /* Buffer dimensions */
+                    { static_cast<int>(points.size()), 3 },  /* Buffer dimensions */
```

Comment:
is this supposed to be ssize_t?

Reply:
It does not compile with `static_cast<ssize_t>